### ISSUE TITLE
Curly brace spacing

### ISSUE THREAD

**Issue by timkurvers** [hyperoslo/eslint-config#3]
The [object-curly-spacing](http://eslint.org/docs/rules/object-curly-spacing) controls the whitespace between the curly braces.

This goes for (nested) object literals on a single line:

``` javascript
const person = { name: 'tim', age: 27, friend: { name: 'bob' } };
```

As well as destructuring:

``` javascript
const { name, age, friend } = person;
```

As of right now, our configuration does not seem to enforce anything. Should we?

**Comment by sindrenm** [comment]
IMO, we definitely should prefer one over the other, as to keep our codebases clean and consistent. Personally, I prefer the spaced version, both for object curlies and destructuring. It's just more pleasant to look at. :smiley: 

Also, if we do it for objects, should we have the same rules for arrays?

``` javascript
const bodybuilders = [ 'Rich Piana', 'Arnold Schwarzenegger' ];
```

**Comment by timkurvers** [comment]
My personal preference is for some reason highly inconsistent:

``` javascript
// Object literals: yes
const person = { name: 'tim', age: 27, friend: { name: 'bob' } };

// Destructuring: no
const {name, age, friend} = person;

// Array literals: no
const bodybuilders = ['Rich Piana', 'Arnold Schwarzenegger'];
```

:disappointed:

**Comment by timkurvers** [comment]
Although that causes some inconsistencies with the following looking super cramped:

``` javascript
function greet({name, age}) {
  console.log(`${name}, such age of ${age}`);
}

greet({ name: 'tim', age: 27 });
```

Perhaps always using spaces for objects and none for arrays?

**Comment by sindrenm** [comment]
Closing this, as it's merged in #4.
